docker-compose-base.yaml

```diff
@@ -1,11 +1,43 @@
 services:
-  app:
+  wishlist:
+    restart: unless-stopped
     build:
       context: .
       dockerfile: base-dockerfile
+    healthcheck:
+      test: wget http://localhost:8080/graphiql || exit 1
+      interval: 1s
+      timeout: 10s
+      retries: 20
+      start_period: 3s
     depends_on:
-      - db
-    ports:
-      - 8000:8000
-  db:
+      wishlist-db:
+        condition: service_healthy
+  wishlist-db:
     image: mongo
+    volumes:
+      - wishlist-db-data:/data/db
+    healthcheck:
+      test: echo 'db.runCommand("ping").ok' | mongosh localhost:27017/test --quiet
+      interval: 10s
+      timeout: 5s
+      retries: 3
+  wishlist-dapr:
+    image: "daprio/daprd:edge"
+    command:
+      [
+        "./daprd",
+        "--app-id",
+        "wishlist",
+        "--app-port",
+        "8080",
+        "--dapr-http-port",
+        "3500",
+        "-placement-host-address",
+        "placement:50006",
+        "--config",
+        "/config.yaml",
```

Comment:
could you add also the resources path?
see https://github.com/MiSArch/catalog/blob/main/docker-compose-base.yaml#L43
this will be required for dapr events